webservices/schemas.py

```diff
@@ -941,5 +969,140 @@ class TotalsCommitteeSchema(schemas['CommitteeHistorySchema']):
 register_schema(ScheduleAByStateRecipientTotalsSchema)
 register_schema(ScheduleAByStateRecipientTotalsPageSchema)
 
+
+# endpoint audit-primary-category
+PrimaryCategorySchema = make_schema(
+    models.PrimaryCategory,
+    fields={
+        'primary_category_id': ma.fields.Int(),
+        'primary_category_name': ma.fields.Str(),
+        'tier': ma.fields.Int(),
+    },
+    options={
+        'exclude': ('tier',)
+    }
+)
+
+PrimaryCategoryPageSchema = make_page_schema(PrimaryCategorySchema)
+register_schema(PrimaryCategorySchema)
+register_schema(PrimaryCategoryPageSchema)
+
+# endpoint audit-category(with nested sub category)
+CategoryRelationSchema = make_schema(
+    models.CategoryRelation,
+    fields={
+        'primary_category_id': ma.fields.Int(),
+        'sub_category_id': ma.fields.Int(),
+        'sub_category_name': ma.fields.Str(),
+        'primary_category_name': ma.fields.Str(),
+    },
+    options={
+        'exclude': ('primary_category_id', 'primary_category_name')
+    }
+)
+
+CategoryRelationPageSchema = make_page_schema(CategoryRelationSchema)
+register_schema(CategoryRelationSchema)
+register_schema(CategoryRelationPageSchema)
+
+# endpoint audit-category
+CategorySchema = make_schema(
+    models.Category,
+    fields={
+        'primary_category_id': ma.fields.Int(),
+        'primary_category_name': ma.fields.Str(),
+        'tier': ma.fields.Int(),
+        'sub_category_list': ma.fields.Nested(CategoryRelationSchema, many=True),
+    },
+    options={
+        'exclude': ('tier',)
+        # 'relationships': [
+        #     Relationship(
+        #         models.Category.sub_category_list,
+        #         models.CategoryRelation.primary_category_id,
+        #         'primary_category_id',
+        #         1
+        #     ),
+        # ],
```

Comment:
Same note as above regarding commented out lines.

However, in this case, we're noting that we may come back to this in the future as a way of solving the problem with retrieving the secondary categories with the primary categories, and/or returning all category information with the main audit case data.